### PR TITLE
feat(web): Add onboarding as the first post-signup

### PR DESCRIPTION
**Pull request by scopsy**
### Screenshots

CleanShot 2024-07-10 at 18 33 51@2x

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
       "skipFiles": [
         "<node_internals>/**"
       ],
-      "type": "pwa-node"
+      "type": "node"
```
Comment (scopsy) [inline .vscode/launch.json:17]: Just some vscode warnings fixed

File: .vscode/tasks.json [hunk 2]
```diff
@@ -10,7 +10,16 @@
...
+        "fileLocation": "autoDetect",
         "owner": "typescript",
+        "pattern": {
+          "regexp": "^([^\\s].*)\\((\\d+|\\d+,\\d+|\\d+,\\d+,\\d+,\\d+)\\):\\s+(error|warning|info)\\s+(TS\\d+)\\s*:\\s*(.*)$",
+          "file": 1,
+          "location": 2,
+          "severity": 3,
+          "code": 4,
+          "message": 5
+        },
```
Comment (scopsy) [inline .vscode/tasks.json:26]: This makes Vscode Tasks more highlighted for reload states

File: apps/web/src/pages/auth/components/QuestionnaireForm.tsx [hunk 3]
```diff
@@ -97,7 +97,19 @@ export function QuestionnaireForm() {
...
+        JobTitleEnum.ARCHITECT,
+        JobTitleEnum.FOUNDER,
+        JobTitleEnum.STUDENT,
+      ].includes(data.jobTitle);
+
+      if (isTechnicalJobTitle) {
+        navigate(ROUTES.DASHBOARD_ONBOARDING);
+      } else {
+        navigate(ROUTES.WORKFLOWS);
+      }
```
Comment (scopsy) [inline apps/web/src/pages/auth/components/QuestionnaireForm.tsx:112]: We only navigate to this page for those job titles

File: apps/web/src/studio/components/GetStartedPageV2/index.tsx [hunk 4]
```diff
@@ -161,6 +198,29 @@ export const GetStartedPageV2 = () => {
                     }}
                   />
                 </div>
+
+                {location === 'onboarding' && (
+                  <SkipCTA
+                    text="No time right now?"
+                    buttonText="Skip Onboarding"
```
Comment (djabarovgeorge) [inline apps/web/src/studio/components/GetStartedPageV2/index.tsx]: ✏ **suggestion:** ‏ 

```suggestion
                    buttonText="Skip Setup"
```

File: libs/novui/package.json [hunk 5]
```diff
@@ -55,6 +55,7 @@
     "./components.css": "./node_modules/@mantine/core/styles.css"
   },
   "scripts": {
+    "start": "npm run build:watch",
```
Comment (djabarovgeorge) [inline libs/novui/package.json:58]: ❓ **question:** ‏ out of curiosity, why do we also need this one?

**Review by djabarovgeorge — APPROVED** [review]
🚀

File: libs/novui/package.json (hunk 5 above)
Comment (scopsy) [reply libs/novui/package.json:58]: Becasue we don't have a "start" command today, and we have it for all of our packages (almost). It's used by webstorm and vscode as a start task comman

**Pull request merged**